lib/active_model/serializer/builder.rb

```diff
@@ -0,0 +1,71 @@
+require "set"
+module ActiveModel
+  class Serializer
+    class Builder
```

Comment:
In fact, I would name this class `Serializer` if there weren't already one... `ResponseSerializer`? `FinalSerializer`? `SerializableResource`?